lib/documents/schemas/life_saving_maritime_appliance_service_stations.json

```diff
@@ -0,0 +1,334 @@
+{
+  "pre_production": true,
+  "content_id": "cfc808eb-1c34-49d1-bfbd-e14dcb2c6bbe",
+  "base_path": "/service-life-saving-maritime-appliances",
+  "format_name": "Find a service station for your inflatable life-saving appliances (format)",
```

Comment:
@ryanb-gds we left this one with the format on it to see where it may surface - no idea 😅